src/plugins/hotkeys/README.md

```diff
@@ -0,0 +1,53 @@
+# Plugin: Hotkeys
+
+Supports for hotkeys to add shortcuts.
+
+## Overview
+
+| <!-- -->         | <!-- -->                                     |
+| ---------------- | -------------------------------------------- |
+| Name             | `pip`                                        |
+| Path             | `vlitejs/plugins/hotkeys`                 |
+| Entry point      | `vlitejs/plugins/hotkeys/hotkeys.js`   |
+| Provider&sup2;   | `'html5', 'youtube', 'vimeo', 'dailymotion'` |
+| Media type&sup3; | `'video', 'audio'`                           |
+
+## Hotkeys
+
+The player accepts the following keyboard shortcuts.
+
+|        Key        | Action                   |
+| :---------------: | ------------------------ |
+| <kbd>space</kbd>  | Toggle playback          |
+|  <kbd>Esc</kbd>   | Exit the fullscreen      |
+| <kbd>&larr;</kbd> | Seek backward of `5s`    |
+| <kbd>&rarr;</kbd> | Seek forward of `5s`     |
+| <kbd>&uarr;</kbd> | Increase volume of `10%` |
+| <kbd>&darr;</kbd> | Decrease volume of `10%` |
```

Comment:
Move this part in the intro, before the overview section. Its part of the plugin description